platform/security/common/src/main/java/org/codice/ddf/security/common/Security.java

```diff
@@ -288,14 +315,42 @@ public Date getExpires(Subject subject) {
    * @return reference to the {@link SecurityManager}
    */
   public SecurityManager getSecurityManager() {
```

Comment:
❗️ We should add `@Nullable` on the return type since this could still return `null`.